chia/cmds/beta_funcs.py

```diff
@@ -0,0 +1,130 @@
+import os
+import sys
+import zipfile
+from datetime import datetime
+from pathlib import Path
+from typing import Any, Callable, Dict, List, Optional
+
+from chia.util.chia_logging import get_beta_logging_config
+from chia.util.config import lock_and_load_config, save_config
+from chia.util.misc import format_bytes, prompt_yes_no
+
+
+def warn_if_beta(config: Dict[str, Any]) -> None:
+    if "beta" in config:
```

Comment:
i think i agree with @altendky here,  whats the benefit of using  this as apposed to an enable/disable flag ?

if your worried about updating the conf then a missing beta can mean False and if it is in the config we can check enabled/disabled